doc/field_guide__eclipsing.md

```diff
@@ -0,0 +1,32 @@
+## Eclipsing variables (ecl)
+
+Eclipsing variables are binary stars systems in which the orbital plane lies nearly in the line of sight of the observer. The binary components pass in front of one another from observer's perspective and so their combined brightness decreases periodically.

```

Comment:
binary stars systems -> binary star systems
from observer's perspective -> from the observer's perspective